sccm/core/plan-design/configs/size-and-scale-numbers.md

```diff
@@ -177,13 +177,7 @@ A central administration site supports a total number of devices that includes u
 
 - 25,000 devices that run Mac and Windows CE 7.0  
 
-- One of the following, depending on how your deployment supports mobile device management (MDM):  
-
-    - 100,000 devices that you manage by using on-premises MDM  
-
-    - 300,000 cloud-based devices  
-
-For example, in a hierarchy you can support 700,000 desktops, up to 25,000 Mac and Windows CE 7.0 devices, and up to 300,000 cloud-based devices when you integrate Microsoft Intune. This hierarchy supports a total of 1,025,000 devices. If you support devices that are managed by on-premises MDM, the total for this hierarchy is 825,000 devices.  
```

Comment:
I suggest keeping the example, just editing out the cloud-based devices part (and then adjusting the math accordingly 😉)